oracles/hardCodedOracle.ts

```diff
@@ -0,0 +1,34 @@
+import { ContractInterface, ethers as ethersCore } from 'ethers';
+import { ethers } from "hardhat";
+import storyDataSourceABI from './poolInfo/abi/storyprotocol/story_data_source.json';
+
+async function main() {
+    const storyDataSourceContractAddress = '0xbEf2D967690A1e354c15E49C1841Ea2C09150E22' // the contract address in goerli testnet
+    const signerWallet = '0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX' // replace with your own wallet address here
+    const hardcodedData = 'Story Data'
```

Comment:
Hardcode data is good enough for PoC. One enhancement we might consider in the future is that retrieves data from the URL specified in the events.

Reply:
Yes. Or we can only allow users to specify certain types of data to fetch.